src/peergos/server/tests/UserTests.java

```diff
@@ -264,6 +264,23 @@ public void changeLoginAlgorithm() throws Exception {
         PeergosNetworkUtils.ensureSignedUp(username, password, network, crypto);
     }
 
+    @Test
+    public void ensureUserIsTryingToFollowPeergos() {
+        String username = generateUsername();
+        String password = "password";
+        UserContext userContext = PeergosNetworkUtils.ensureSignedUp(username, password, network, crypto);
+        Boolean requestToFollowPeergos = userContext.getSocialState().join().pendingOutgoingFollowRequests.containsKey("peergos");
+        assertTrue("Pending follow request Peergos exists after sign-up", requestToFollowPeergos);
+        userContext.unfollow("peergos");
+        requestToFollowPeergos = userContext.getFollowing().join().contains("peergos");
```

Comment:
This will never be true without interaction from the peergos user

Reply:
"I would log in as peergos and accept the request, then get the user to remove peergos as a follower